README.md

```diff
@@ -27,6 +27,16 @@ npm start
 
 Now open [localhost:8080](http://localhost:8080)
 
+## Offline First
+
+This tutorial implments ServiceWorker and can be run offline using a Chrome or Firefox browser. To run offline, turn off the server (Ctrl + C) and refresh [localhost:8080](http://localhost:8080) using Chrome or Firefox. The tutorial page will still load.
+
+To remove the offline tutorial from [localhost:8080](http://localhost:8080) visit chrome://serviceworker-internals/ if using Chrome or visit about:serviceworkers if using Firefox and unregister the ServiceWorker. 
```

Comment:
@gr2m Edge is currently [in development](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/serviceworker) for Service Worker.

Opera browser supports Service Worker and will add instructions to unregister.